<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBInstaller</class>
 <widget class="QMainWindow" name="OpenRGBInstaller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenRGB-Installer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <item row="3" column="1">
     <widget class="QComboBox" name="versionComboBox"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Version to install</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Installed version</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QProgressBar" name="statusProgressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="format">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="installedVersionLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="uninstallButton">
      <property name="text">
       <string>Uninstall</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="launchButton">
      <property name="text">
       <string>Launch OpenRGB</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="installButton">
      <property name="text">
       <string>Install</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
